apps/backoffice-v2/src/common/components/atoms/Welcome/Welcome.tsx

```diff
@@ -0,0 +1,25 @@
+import React, { FunctionComponent, useMemo } from 'react';
+import { t } from 'i18next';
+import { useAuthenticatedUserQuery } from '../../../../domains/auth/hooks/queries/useAuthenticatedUserQuery/useAuthenticatedUserQuery';
+import { UserAvatar } from '../../atoms/UserAvatar/UserAvatar';
+
+export const Welcome: FunctionComponent = () => {
+  const { data: session } = useAuthenticatedUserQuery();
+  const user = useMemo(
+    () => ({
+      firstname: session?.user?.firstName,
+      fullname: `${session?.user?.firstName} ${session?.user?.lastName}`,
+      avatarUrl: session?.user?.avatarUrl,
+    }),
+    [session?.user?.firstName, session?.user?.lastName, session?.user?.avatarUrl],
+  );
+  return (
```

Comment:
Separating the different lifecycle stages of a function using a linebreak or different groups of related logic would help readability. Here the stages are 1. variable declaration/business logic 2. the render function (return statement)
```suggestion
  );
  
  return (
```